### PR TITLE
Print version

### LINKED ISSUE #15: Add version number
It can be hard to know if using `pip` to update the cli actually worked. Sometimes it seems necessary to uninstall and reinstall to get the updates. Something that would help with this would be to introduce a versioning scheme that makes it clear if the library has been updated. Would like to be able to run `aerie-cli version` or `aerie-cli -v` or something

### PR DESCRIPTION
**Pull request by cartermak**
Closes #15

### CODE REVIEW

File: src/aerie_cli/__main__.py [hunk 1]
```diff
@@ -23,19 +25,40 @@
...
+
+
 @app.callback()
-def setupGlobalCommandContext(
-        hasura_admin_secret = typer.Option(default="",
-                                           help="Hasura admin secret that will be put in the header of graphql requests")
-    ):
+def app_callback(
+    version: Optional[bool] = typer.Option(
+        None,
```
Comment (Mythicaeda) [inline src/aerie_cli/__main__.py:38]: Why not provide `-v` as an option?

**Review by Mythicaeda — APPROVED** [review]
It worked for me on Python 3.11

File: src/aerie_cli/__main__.py (hunk 1 above)
Comment (cartermak) [reply src/aerie_cli/__main__.py:38]: Ope good catch, I will add that back in. I had it at one point, but this is, like, rev 5 of how to implement this and I lost it.

**Review by Navarro-Jonathan — APPROVED** [review]
After testing locally this seems to work as intended. The change to the app callback's name make that part a little more clear too. Everything looks good to me.

**Pull request merged**